packages/frontend/hooks/src/use-block-suite-editor.ts

```diff
@@ -0,0 +1,30 @@
+import type { AffineEditorContainer } from '@blocksuite/presets';
+import { atom, useAtom } from 'jotai';
+import { useCallback, useMemo } from 'react';
+
+export const editorContainerAtom = atom<{
```

Comment:
I think if you expose hooks, there is no point of exposing the atom any longer.
Also the API does not consider the case that two editors share the same pageId